sdcm/cluster.py

```diff
@@ -2654,6 +2654,11 @@ def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):
         self.log.info('Setup in BaseLoaderSet')
         node.wait_ssh_up(verbose=verbose)
 
+        # fix https://github.com/scylladb/scylla-cluster-tests/issues/1080
+        self.log.debug("enlarge sshd MaxSessions to 100")
+        node.remoter.run(r"sed -i 's/#MaxSessions \(.*\)$/MaxSessions 100/' /etc/ssh/sshd_config", ignore_status=True)
+        node.remoter.reconnect()
```

Comment:
Another solution is updating the config in loader AMI, and 'release' a new version of loader AMI.
Updating config in node_setup of loader will also make loader that is not prepared from our specific AMI works, that's better.

Reply:
I think better solution is to put this to the instance metadata: 
- AWS: UserData 
- GCE:  `startup-script`

Reply:
And what about other backend like Baremetal ? we'll need todo that manually ?

also we are using AWS UserData for passing down Scylla args (in both monitor and db nodes), I.e. this won't be a general solution we could use across all types of nodes.

this was intended as a quick patch, since this issue can bite us in any place we are using lots of stress commands.

Reply:
You can pass `--base64postscript="#!/bin/bash ....."`
Defined in ./scylla-ami/ds2_configure.py
```
    parser.add_argument("--base64postscript", action="store", type=str, dest="base64postscript")
    # Option that allows to download and execute a custom script
```